doc/content/draft-template/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Draft Template"
+description: "This page is a draft template"
+weight: 
+draft: true
+---
+
+Guidelines here are taken from [DEVELOPMENT.md](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/DEVELOPMENT.md), which you should read. This template exists as a copy and paste starting point for new documentation. Remove the `draft`key in the Front Matter to generate the page (otherwise hugo will skip it).
+
+A documentation page starts with an introduction, and then the first heading. The first paragraph of the introduction is typically a summary of the page. Use a <!--more--> to indicate where the summary ends.
+
+<!--more-->
+
+## Requirements
+
+Use a requirements subheading to list requirements/prerequisites.
+
+1. Requirement 1
+2. Requirement 2
+
+## Sections
+
+When possible, divide long documents into separate files, each with its own folder and `_index.md`. Use the `weight`tag in the [Front Matter](https://gohugo.io/content-management/front-matter/) to manually sort sections if necessary. If not, they will be sorted alphabetically.
+
+## Subheadings
+
+Since the title is a h1, everything in the content is at least h2 (##). Use subheadings to break up sections and provide a high level overview of steps the user should follow.
+
+## Notes
+
+Use the `>` markdown code, and include the word note to distinguish it from a code block.
+
+>Note: if you need help with any CLI command, use the `--help` flag to get a list of subcommands, flags and their description and aliases.
```

Comment:
Speaking of which, good to introduce the short codes; `cli-only` and `tts` are used quite a lot